packages/atomic/src/components/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -138,6 +141,7 @@ export class AtomicFoldedResultList implements InitializableComponent {
       templateElements: this.host.querySelectorAll('atomic-result-template'),
       onReady: () => {
         this.ready = true;
+        this.assignRenderingFunctionIfPossible();
```

Comment:
why is this change necessary? it loses the rendering method reference when scrolling?

Might be better to improve the solution with a slightly different architecture rather than have this kind of patch

Reply:
I copied what @olamothe did for the other result list - it's not related to scrolling, it's related to trying to set rendering function before `initialize` happens. currently throws an error saying that `resultListCommon` doesn't exist

Reply:
Yeah, it's mostly about not having to deal with "are components properly initialized yet?" in Atomic react code, because that's kind of tricky to deal with.

So you can assign the render function at any time, and it will be picked up by the Atomic result list when it's ready to do its thing.